esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2972,6 +2972,10 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
         if (!validateDropZonePath(nullptr, netaddr, sPath) && !validateConfigurationDirectory(nullptr, "log", nullptr, nullptr, sPath)) //The path should be the absolute path for the dropzone or log file.
             throw makeStringExceptionV(ECLWATCH_INVALID_INPUT, "Invalid file path %s", sPath.str());
 
+        const char *dropZoneId = isContainerized() ? req.getDropZoneName() : netaddr;
```

Comment:
see prev. comments re. looking up dropzone name.

But I see this ::onFileList method insists on a netaddr : 
```
        const char* netaddr = req.getNetaddr();
        if (!netaddr || !*netaddr)
            throw MakeStringException(ECLWATCH_INVALID_INPUT, "Network address not specified.");
```

that looks wrong ?

Reply:
In bare-metal, one dropzone may have multiple dropzone servers. The ::onFileList method is designed to retrieve dropzone files in one dropzone server.

Reply:
ok.. but what about onFileList in cloud ?

Reply:
In the existing commits of this PR, a DropZoneName is added into the request. This check should be removed since, in cloud, we may find out the host based on the DropZoneName.

Reply:
in cloud it should always be a dropzone name that is supplied, never a host/ip.

Reply:
> This check should be removed since, in cloud, we may find out the host based on the DropZoneName.
@jakesmith I may take it back. In the existing design, the ::onFileList method is designed to retrieve dropzone files in one dropzone host. One DropZone may have >1 hosts. If the netaddr is not specified, we do not know which host's dropzone files should be reported, even in cloud. I am not sure, here, we should report the dropzone files in all of the hosts. To report the dropzone files in all of the hosts, a host layer may add on the top of the files. That will break the existing ECLWatch (the Landing Zone page). ECLWatch has to parse the data, find the files for each host, and pre-populate the data for each host. Or ECLWatch may remove the host layer (not display the files under a host, but, display the files under a dropzone).

Reply:
I guess I can report the dropzone files in all of the hosts if no host is specified.